app/javascript/src/components/Dashboard/Contacts/Constants.jsx

```diff
@@ -0,0 +1,302 @@
+    id: 13,
+  },
+  {
+    name: "Jacob Jones",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: JacobDisplayPicture,
+    id: 14,
+  },
+  {
+    name: "Ronald Rechards",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: "",
+    id: 15,
+  },
+  {
+    name: "Jacob Jones",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: JacobDisplayPicture,
+    id: 16,
+  },
+  {
+    name: "Ronald Rechards",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: "",
+    id: 17,
+  },
+  {
+    name: "Jacob Jones",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: JacobDisplayPicture,
+    id: 18,
+  },
+  {
+    name: "Ronald Rechards",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: "",
+    id: 19,
+  },
+  {
+    name: "Jacob Jones",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: JacobDisplayPicture,
+    id: 20,
+  },
+  {
+    name: "Ronald Rechards",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: "",
+    id: 21,
+  },
+  {
+    name: "Jacob Jones",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: JacobDisplayPicture,
+    id: 22,
+  },
+  {
+    name: "Ronald Rechards",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: "",
+    id: 23,
+  },
+  {
+    name: "Jacob Jones",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: JacobDisplayPicture,
+    id: 24,
+  },
+  {
+    name: "Ronald Rechards",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: "",
+    id: 25,
+  },
+  {
+    name: "Jacob Jones",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: JacobDisplayPicture,
+    id: 26,
+  },
+  {
+    name: "Ronald Rechards",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: "",
+    id: 27,
+  },
+  {
+    name: "Jacob Jones",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: JacobDisplayPicture,
+    id: 28,
+  },
+  {
+    name: "Ronald Rechards",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: "",
+    id: 29,
+  },
+  {
+    name: "Jacob Jones",
+    email: "albert@borer.com",
+    createdAt: "Feb, 5, 2021",
+    icon_button: "",
+    displayPicture: JacobDisplayPicture,
+    id: 30,
+  },
+];
+
+export const INIT_VALUES = {
```

Comment:
Cut shorting names to sound like an abbreviation doesn't add any benefit unless the abbreviation doesn't have multiple meanings.
```suggestion
export const INITIAL_VALUES = {
```